tailcfg/tailcfg.go

```diff
@@ -920,6 +920,12 @@ type PingRequest struct {
 	// Log is whether to log about this ping in the success case.
 	// For failure cases, the client will log regardless.
 	Log bool `json:",omitempty"`
+
+	// Types is the types of ping that is initiated. Can be TSMP, ICMP or DISCO
+	Types string
+
+	// IP is the IP that we will attempt to Ping
```

Comment:
lowercase ping. trailing period.

Also, no "we".  Maybe: "IP is the ping target."

But clarify when this is required. Because for old-style PingRequest, only URL is required.